stdlib/2/os/__init__.pyi

```diff
@@ -8,16 +8,95 @@ from . import path
 from mypy_extensions import NoReturn
 
 error = OSError
+SEEK_SET = 0
+SEEK_CUR = 0
+SEEK_END = 0
+
+# More constants, copied from stdlib/3/os/__init__.pyi
+O_RDONLY = 0
+O_WRONLY = 0
+O_RDWR = 0
+O_APPEND = 0
+O_CREAT = 0
+O_EXCL = 0
+O_TRUNC = 0
+O_DSYNC = 0    # Unix only
+O_RSYNC = 0    # Unix only
+O_SYNC = 0     # Unix only
+O_NDELAY = 0   # Unix only
+O_NONBLOCK = 0  # Unix only
+O_NOCTTY = 0   # Unix only
+O_SHLOCK = 0   # Unix only
+O_EXLOCK = 0   # Unix only
+O_BINARY = 0     # Windows only
+O_NOINHERIT = 0  # Windows only
+O_SHORT_LIVED = 0  # Windows only
+O_TEMPORARY = 0  # Windows only
+O_RANDOM = 0     # Windows only
+O_SEQUENTIAL = 0  # Windows only
+O_TEXT = 0       # Windows only
```

Comment:
I agree.  But this change is mostly a re-ordering. I can do that in a future diff?  This is mostly a reordering to make the files mostly identical and easier to diff/modify.